Source/Analytics/Read/Overview/LastWeekTotals/CaseReportTotalsQuery.cs

```diff
@@ -0,0 +1,19 @@
+using System;
```

Comment:
Should have a header:
```suggestion
/*---------------------------------------------------------------------------------------------
*  Copyright (c) The International Federation of Red Cross and Red Crescent Societies. All rights reserved.
*  Licensed under the MIT License. See LICENSE in the project root for license information.
*--------------------------------------------------------------------------------------------*/

using System;
```